packages/datadog-plugin-mongodb-core/src/index.js

```diff
@@ -120,6 +136,13 @@ function shouldHide (input) {
 }
 
 function limitDepth (input) {
+  if (Array.isArray(input) && input.length > 0) {
+    for (let i = 0; i < input.length; i++) {
+      const child = input[i]
+      input[i] = limitDepth(child)
```

Comment:
Don't need to store `child`. You can just pass `input[i]` to `limitDepth()` directly.